packages/web-components/src/components/cta/card-cta.ts

```diff
@@ -47,6 +47,13 @@ class DDSCardCTA extends VideoCTAMixin(CTAMixin(DDSCard)) {
       return super._renderHeading();
     }
     const caption = formatVideoCaptionInEffect({ name: videoName });
+
+    this.dispatchEvent(
+      new CustomEvent('video-updated', {
```

Comment:
Works great! Let's just update the custom event name to something like `dds-card-cta-video-title-updated` and use getters instead of a string, both here and in the `carousel.ts` file like we did for the other issue.

Reply:
Great suggestion! Please see the changes here (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8336/commits/a057f6519ec806c246f94e2dc6176f084c5afb0b).